### ISSUE TITLE
"Site connection and user authentication"

### ISSUE THREAD

**Issue by eclarke1** [google/site-kit-wp#3448]
## Feature Description

- Should explain the concept of “Site connection” and how it differs from “user authentication” (e.g. while the two processes are partly merged, only once does a site need to connect, while then every user needs to individually authenticate)
- Should also explain “site reset” (undoes site connection and everything else) vs “user disconnect” (undoes only the current user authentication and related data)
- Should also explain granting additional scopes after already being authenticated
-- New read scopes, e.g. when activating a module
-- New write scopes (on demand), e.g. when wanting to edit configuration of a Google entity
- Primary focus is to clarify the terminology used for these very closely related functionalities
- Should not focus on explaining Site Kit Service

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The ["Site connection and user authentication" article](https://github.com/google/site-kit-wp/blob/develop/docs/Site-connection-and-user-authentication.md) should be written.
* It should holistically outlined the different terminology in Site Kit between sites, users, and modules, but specifically (as part of that) cover the bullet points in the issue description above.
* It should follow the [contributor documentation best practices](https://docs.google.com/document/d/1uiLfbH6EUsOBryjHd3HdhRnwjNM6iw4-ug5Z183mduw/edit#heading=h.mxlvului4mhv). Please re-read them before starting to write.
    * The article should have roughly between 500-2000 words - no hard limits though. Since this is more of an introduction to a topic, it can likely stay below 1000.
    * Also before writing, please review the list of all (eventual) articles in the [TOC](https://github.com/google/site-kit-wp/blob/develop/docs/README.md) - if you refer to anything that is closely related to an article heading there, or if you refer to something that "undoubtedly" will be covered in a specific other article there, please includes links to such articles, even if they aren't written yet.
    * All links to other articles should be relative (i.e. work in the current branch you're in).
* The "(TODO)" annotation in the article itself and in the [TOC](https://github.com/google/site-kit-wp/blob/develop/docs/README.md) should be removed.

## Implementation Brief

**Instead of a regular IB, please provide your proposed document outline (nested list of headings) for the article here. Use headings that the article could _actually_ use, not just some brainstorm type of preliminary heading.** Please assign to @felixarntz for IB review, specifically since this is one of the first articles in the epic.

* 

### Test Coverage

* 

### Visual Regression Changes

* 

## QA Brief

* 

## Changelog entry

*

**Comment by eclarke1** [comment]
@felixarntz @aaemnnosttv @marrrmarrr this was one of the 4 articles voted highest priority by engineers so if possible to progress this through the workflow along with the others labelled 'Type: Docs' and have P1 assigned.

**Comment by aaemnnosttv** [comment]
Closing in favor of #10620
